### PR TITLE
Feature/amenities crud

### PR DESCRIPTION
**Pull request by facamesi52**

### CODE REVIEW

**Review by viamazoba — APPROVED** [review]

File: src/api/hotel/hotel.service.ts [hunk 1]
```diff
@@ -41,34 +52,64 @@ export async function getHotelById(id: string) {
...
+    //           include: {country: true}},
+    //         Service_labels_hotel: {
+    //           include: {
+    //             label: true
+    // >>>>>>> dev
+    //           }
+    //         }
+    //       },
+    //     });
+    // const serviceLabels = hotel?.Service_labels_hotel.map(item => item.label);
```
Comment (ht1204) [inline src/api/hotel/hotel.service.ts:105]: Please, delete commented code

File: src/api/hotel/hotel.service.ts [hunk 2]
```diff
@@ -41,34 +52,64 @@ export async function getHotelById(id: string) {
...
+    return hotel
+    //  {
 
-    return {
-      ...hotel,
-      Service_labels_hotel: serviceLabels,
-    };
+    //   // ...hotel,
+    //   // Service_labels_hotel: serviceLabels,
+    // };
```
Comment (ht1204) [inline src/api/hotel/hotel.service.ts:112]: AND here

File: src/api/room/index.ts [hunk 3]
```diff
@@ -9,10 +9,10 @@ import  { hasRole, isAuthenticated }  from "../../auth/auth.controller";
...
 const router= Router();
 
-router.post('/',isAuthenticated, hasRole(['admin']), createRoomHandler);
+router.post('/',/*isAuthenticated, hasRole(['admin']),*/ createRoomHandler);
 router.get('/', getRoomController);
 router.get('/:id', getRoomByIdController);
-router.put('/:id', isAuthenticated, hasRole(['admin']),updateRoomController);
-router.delete('/:id', isAuthenticated, hasRole(['admin']), deleteRoomController);
+router.put('/:id', /*isAuthenticated, hasRole(['admin']),*/updateRoomController);
+router.delete('/:id', /*isAuthenticated, hasRole(['admin']),*/ deleteRoomController);
```
Comment (ht1204) [inline src/api/room/index.ts:16]: Guess this Will be changed or removed

**Review by ht1204 — APPROVED** [review]
Approve, but please, address the comments before merge

**Review by Nataliachm — APPROVED** [review]

**Pull request merged**